tests/model_hub_tests/torch_tests/hf_transformers_models

```diff
@@ -0,0 +1,411 @@
+# List of models obtained from huggingface_hub api as one model per architecture with highest likes and downloads
+#abcp4/mymodel-test,mymodel
```

Comment:
I propose to xfail them and create a mark xfail. This way we can track regressions and pregress more effective